gapis/api/vulkan/api/enums.api

```diff
@@ -91,6 +91,7 @@ enum VkResult {
 }
 
 /// Structure type enumerant
+@analyze_usage
```

Comment:
Is there any reason you are specifically keeping analyze_usage on this enum (and the next)?

Reply:
They have duplicate values (both have some labels that have <label> and <label>_KHR versions), and so as is constant_sets.go will panic when it encounters them.  I'm open to better solutions to enums with duplicates though.

Reply:
That's fair. This should be ok for now. Those 2 enums are cases where we probably SHOULD be analyzing usage anyway.